notebooks/llm-chatbot/rag-chatbot.ipynb

```diff
@@ -18,25 +18,31 @@
     "- Download and convert the model from a public source using the [OpenVINO integration with Hugging Face Optimum](https://huggingface.co/blog/openvino).\n",
```

Comment:
<p>what the reason to use strict numpy version?</p><p>We avoiding strict dependencies because they may be not available for some new python versions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1848/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I found there is some error message when using higher version of numpy, and i wll validate it again

Reply:
removed